library/src/main/java/com/meisolsson/githubsdk/core/FormattedTimeAdapter.java

```diff
@@ -30,8 +30,14 @@
     @FromJson
     @FormattedTime
     Date fromJson(String time){
-        DateTimeFormatter format = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
-        DateTime t = format.withZoneUTC().parseDateTime(time);
+        DateTime t;
+        try {
+            DateTimeFormatter format = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
+            t = format.withZoneUTC().parseDateTime(time);
+        } catch (IllegalArgumentException e) {
+            DateTimeFormatter format = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZ");
+            t = format.withOffsetParsed().parseDateTime(time);
+        }
```

Comment:
I'll probably ask GitHub support about whether the created_at field of the block field in the error response can be made compliant with all other aspects of the API.

Reply:
I should follow up when them on this. Last time they said they'll investigate.